docs/src/modules/components/DemoToolbar.js

```diff
@@ -109,36 +108,30 @@ const Button = styled(MDButton)(({ theme }) => ({
   flexShrink: 0,
   borderRadius: 999,
   border: 'none',
-  paddingLeft: theme.spacing(1),
-  paddingRight: theme.spacing(0.5),
   fontSize: theme.typography.pxToRem(13),
   fontWeight: theme.typography.fontWeightMedium,
-  color: theme.palette.primary.main,
+  color: theme.palette.text.secondary,
   '& svg': {
-    fill: theme.palette.primary.main,
+    color: theme.palette.text.secondary,
```

Comment:
From what I see in the preview, this rule is not applied. It's overridden by another rule defined in this file:
```
'& .MuiSvgIcon-root': {
      fontSize: 16,
      color: (theme.vars || theme).palette.grey[800],
    },
```

'& .MuiSvgIcon-root' -> selector with 2 classes
'& .svg' -> selector with 1 classes + 1 type

the first one wins since class are more [selective](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) than type

Reply:
I think I messed up this one ⎯ it was `fill` and then I changed it to `color` 😬. I guess the former is correct, right?

Reply:
I think this color has never been applied

https://github.com/mui/material-ui/assets/45398769/84246f84-a916-49a9-8577-52a7c4286e74

I think you can remove this `'& svg'
And delete the color applied in `'& .MuiSvgIcon-root': {`

It should do the job since all icons are in a button. Before it was not the case because of the toggle controlling code expansion

Reply:
Gotcha! Removed it 🤙